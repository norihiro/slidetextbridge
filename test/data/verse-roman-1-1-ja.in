キリスト・イエスのしもべ、神の福音のために選び出され、使徒として召されたパウロから。

──この福音は、神がご自分の預言者たちを通して、聖書にあらかじめ約束されたもので、

御子に関するものです。御子は、肉によればダビデの子孫から生まれ、

聖なる霊によれば、死者の中からの復活により、力ある神の子として公に示された方、私たちの主イエス・キリストです。

この方によって、私たちは恵みと使徒の務めを受けました。御名のために、すべての異邦人の中に信仰の従順をもたらすためです。

その異邦人たちの中にあって、あなたがたも召されてイエス・キリストのものとなりました──

ローマにいるすべての、神に愛され、召された聖徒たちへ。私たちの父なる神と主イエス・キリストから、恵みと平安があなたがたにありますように。

