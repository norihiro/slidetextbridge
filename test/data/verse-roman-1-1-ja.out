キリスト・イエスのしもべ、神の福音のた
めに選び出され、使徒として召されたパウ
ロから。

──この福音は、神がご自分の預言者たちを
通して、聖書にあらかじめ約束されたもの
で、

御子に関するものです。御子は、肉によれ
ばダビデの子孫から生まれ、

聖なる霊によれば、死者の中からの復活に
より、力ある神の子として公に示された方、
私たちの主イエス・キリストです。

この方によって、私たちは恵みと使徒の務
めを受けました。御名のために、すべての
異邦人の中に信仰の従順をもたらすためで
す。

その異邦人たちの中にあって、あなたがた
も召されてイエス・キリストのものとなり
ました──

ローマにいるすべての、神に愛され、召さ
れた聖徒たちへ。私たちの父なる神と主イ
エス・キリストから、恵みと平安があなた
がたにありますように。

